docs/Developers/Development_Setup.rst

```diff
@@ -39,7 +39,13 @@ master branch from and all of its dependencies with pip using
 
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala-bears
+    (venv)$ pip3 install .
```

Comment:
Ooh. I think I got it.
Do you want me to change the above _pip3_ lines with `pip3 install -e . ` or do you want me to create a paragraph below in which the user will be informed that he needs to add the `-e` flag in order to editable install **coala** from repository?